index.js

```diff
@@ -43,10 +43,11 @@ module.exports = class handlePeers extends EventEmitter {
       let peers = peerBookToJson(node.peerBook)
 
       if (Object.keys(peers).length === 0) {
-        p.push(Buffer.from([0]))
+        p.push(Buffer.from(Uint32Array.from([0]).buffer))
       } else {
         peers = Buffer.from(JSON.stringify(peers))
-        p.push(Buffer.from([peers.length]))
+        const len = Uint32Array.from([peers.length]);
```

Comment:
it would be best if we used [leb128](https://github.com/wanderer/pull-leb128) to encoded, to remain consistent. you should be able to do something like this  `leb.unsigned.encode(peers.length, p)`